src/main/java/com/example/ticketbooking/model/PriceCalculator.java

```diff
@@ -0,0 +1,30 @@
+package com.example.ticketbooking.model;
+
+import com.example.ticketbooking.model.request.ReservationRequest;
+import lombok.experimental.UtilityClass;
+
+import java.util.Set;
+
+@UtilityClass
+public class PriceCalculator {
+
+    public static double calculate(Set<ReservationRequest.TicketDetails> ticketType) {
+        return ticketType.stream()
+                .mapToDouble(t -> getTicketPrice(t.getTicketType()))
+                .sum();
+    }
+
+    private static double getTicketPrice(TicketType ticketType) {
```

Comment:
A co jeśli zmienią się wymagania odnośnie cen biletów? :) Da się to zrobić inaczej, żeby nie było potrzeby ponownego budowania i wdrażania całej aplikacji?

Reply:
racja, lepiej byłoby te wartości przechowywać/zmieniać w DB jakimś serwisem/kontrolerem :D